Pod/Classes/PINRemoteImageManagerResult.h

```diff
@@ -7,8 +7,13 @@
 //
 
 @import UIKit;
-
+#if __has_include(<FLAnimatedImage/FLAnimatedImage.h>)
 #import <FLAnimatedImage/FLAnimatedImage.h>
+#define USE_FLANIMATED_IMAGE    1
+#else
+#define USE_FLANIMATED_IMAGE    0
+#define FLAnimatedImage         NSObject
```

Comment:
Ah, interesting. I think this might also be a good opportunity to clean up that alias, too. We probably want to apply the `USE_FLANIMATED_IMAGE` condition to all FLAnimatedImage references throughout the library.

I'm not very familiar with the FLAnimatedImage component, however, so let's see what @garrettmoon has to say.